src/main/java/org/broadinstitute/hellbender/utils/read/markduplicates/sparkrecords/Passthrough.java

```diff
@@ -0,0 +1,28 @@
+package org.broadinstitute.hellbender.utils.read.markduplicates.sparkrecords;
+
+import htsjdk.samtools.SAMFileHeader;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.utils.read.markduplicates.ReadsKey;
+
+/**
+ * Dummy class used for preserving reads that need to be marked as non-duplicate despite not wanting to perform any
+ * processing on the reads. (eg. unmapped reads we don't want to process but must be non-duplicate marked)
+ */
+public final class Passthrough extends MarkDuplicatesSparkRecord {
+    private final transient GATKRead read;
+
+    Passthrough(GATKRead read, int partitionIndex) {
+        super(partitionIndex, read.getName());
```

Comment:
Should make the key at construction time instead of holding a transient reference to the read (can do in a separate PR, however).

Reply:
done